### PR TITLE
VIH-7961 remove smoke check

### PR DESCRIPTION
**Pull request by lowflying**
### JIRA link (if applicable) ###

VIH-7961

### Change description ###

Added some logic to allow toggling of the Smoke Tests 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: jobs/azWebAppDeployJob.yml [hunk 1]
```diff
@@ -232,6 +247,12 @@ jobs:
           steps:
           - checkout: self
 
+          - task: Bash@3
```
Comment (chris-pateman) [inline jobs/azWebAppDeployJob.yml]: As this is within the if statement would it not just log it as true every time?
Also there is a bash log of this parameter further up already

File: jobs/azWebAppDeployJob.yml [hunk 2]
```diff
@@ -192,6 +192,21 @@ jobs:
...
+        - task: AzureRmWebAppDeployment@4
+          name: azureDeployment
+          displayName: Deploy to ${{ parameters.slot }} Slot
+          inputs:
+            azureSubscription: ${{ parameters.subscription }}
+            WebAppName: ${{ parameters.appName }}
+            deployToSlotOrASE: true
+            ResourceGroupName: ${{ parameters.resourceGroup }}
+            SlotName: ${{ parameters.slot }}
+            packageForLinux:  $(Pipeline.Workspace)/App/App.zip
```
Comment (chris-pateman) [inline jobs/azWebAppDeployJob.yml:205]: should this be a variable to make sure it is consistent through out the template as I have seen it in few places?

File: jobs/azWebAppDeployJob.yml [hunk 3]
```diff
@@ -232,6 +247,12 @@ jobs:
           steps:
           - checkout: self
```
Comment (chris-pateman) [inline jobs/azWebAppDeployJob.yml]: no exactly part of this PR, but the AC test steps after, there seems to be a lot of them. Would it be worth splitting them out to another template, so we can just reference 'template: azWebAppAcceptanceTests.yml' ?

File: jobs/azWebAppDeployJob.yml (hunk 2 above)
Comment (lowflying) [reply jobs/azWebAppDeployJob.yml]: Variable'd now

File: jobs/azWebAppDeployJob.yml (hunk 3 above)
Comment (lowflying) [reply jobs/azWebAppDeployJob.yml]: A good idea, there's a decent amount that can be refactored. However the AC tests use a decent amount of parameters that are unpacked just above, I'll create a ticket for this but don't think it's needed for this PR

File: jobs/azWebAppDeployJob.yml [hunk 4]
```diff
@@ -94,6 +90,11 @@ parameters:
     type: string
     default: ""
 
+  - name: runACTests
```
Comment (PrasannaDommalapati) [inline jobs/azWebAppDeployJob.yml:93]: this is not required in parameters as we have it in variables.

File: jobs/azWebAppDeployJob.yml [hunk 5]
```diff
@@ -192,6 +195,23 @@ jobs:
...
+          displayName: Deploy to ${{ parameters.slot }} Slot
+          inputs:
+            azureSubscription: ${{ parameters.subscription }}
+            WebAppName: ${{ parameters.appName }}
+            deployToSlotOrASE: true
+            ResourceGroupName: ${{ parameters.resourceGroup }}
+            SlotName: ${{ parameters.slot }}
+            packageForLinux:  ${{ variables.appPackagePath }}
+
+        - task: Bash@3
```
Comment (PrasannaDommalapati) [inline jobs/azWebAppDeployJob.yml:210]: This need to be removed?

Comment (lowflying) [reply jobs/azWebAppDeployJob.yml:210]: it does!

**Review by lowflying — COMMENTED** [review]
removed logging

**Review by PrasannaDommalapati — APPROVED** [review]
LGTM

**Pull request merged**